spec/features/admin/referrers/referrals_list_toggle_spec.rb

```diff
@@ -22,16 +24,17 @@
     click_link referrer1.email
     expect(page).to have_content referral1_email
 
+    sleep 1
```

Comment:
Lets use some method from capybara which waits for JS/Ajax events, instead of sleep.